tests/factories.py

```diff
@@ -285,6 +251,39 @@ class Params:
     navlink = factory.SubFactory(NavlinkFactory)
 
 
+class ActionReportFactory(
+    factory.Factory, metaclass=BaseMetaFactory[types_.ActionReport]  # type: ignore[misc]
+):
+    """Generate Action reports."""  # noqa: DCO060
```

Comment:
Would it be good to mention DCO060 is from `flake8-docstrings-complete`?

Reply:
I think at the top of this file it is extensively documented why these checks are disabled here. I'll see if it can be done at the file level actually

Reply:
I'll do this in a follow up PR